test-resources/kt/feature_flag_system_2/control/input/XPMethodChainCases.kt

```diff
@@ -47,6 +47,14 @@ internal class XPMethodChainCases {
             if (sp.isOtherFlag().cachedValue || sp.isStaleFeature().cachedValue) {
                 println("!!!")
             }
+            // simple_identifier && false, should be deleted
+            if (a && sp.isStaleFeature().cachedValue){
+                println("!!!")
```

Comment:
can we have different strings in the `println` ? So that its easier to figure out the clenaup?